docs/get-started/control-motor.md

```diff
@@ -244,6 +244,197 @@ func main() {
+  /// the Smart Machine, but is not actually connected to the machine itself.
+  ///
+  /// This is initialized late because it requires an asynchronous
+  /// network call to establish the connection.
+  late RobotClient client;
+
+  @override
+  void initState() {
+    super.initState();
+    // Call our own _initState method to initialize our state.
+    _initState();
+  }
+
+  @override
+  void dispose() {
+    // You should always close the [RobotClient] to free up resources.
+    // Calling [RobotClient.close] will clean up any tasks and
+    // resources created by Viam.
+    client.close();
+    super.dispose();
+  }
+
+  /// This method will get called when the widget initializes its state.
+  /// It exists outside the overridden [initState] function since it's async.
+  Future<void> _initState() async {
+    // Using the authenticated [Viam] the received as a parameter,
+    // the app can obtain a connection to the Robot.
+    // There is a helpful convenience method on the [Viam] instance for this.
+    final robotClient = await widget._viam.getRobotClient(widget.robot);
+    setState(() {
+      client = robotClient;
+      _isLoading = false;
+    });
+  }
+
+  /// A computed variable that returns the available [ResourceName]s of
+  /// this robot in an alphabetically sorted list.
+  List<ResourceName> get _sortedResourceNames {
+    return client.resourceNames..sort((a, b) => a.name.compareTo(b.name));
+  }
+
+  bool _isNavigable(ResourceName rn) {
+    if (rn.subtype == Motor.subtype.resourceSubtype) {
+      return true;
+    }
+    return false;
+  }
+
+  void _navigate(ResourceName rn) {
+    if (rn.subtype == Motor.subtype.resourceSubtype) {
+      final motor = Motor.fromRobot(client, rn.name);
+      Navigator.of(context).push(MaterialPageRoute(builder: (_) => MotorScreen(motor)));
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text(widget.robot.name)),
+        body: _isLoading
+            ? const Center(child: CircularProgressIndicator.adaptive())
+            : ListView.builder(
+                itemCount: client.resourceNames.length,
+                itemBuilder: (_, index) {
+                  final resourceName = _sortedResourceNames[index];
+                  return ListTile(
+                    title: Text(resourceName.name),
+                    subtitle: Text(
+                        '${resourceName.namespace}:${resourceName.type}:${resourceName.subtype}'),
+                    onTap: () => _navigate(resourceName),
+                    trailing: _isNavigable(resourceName) ? Icon(Icons.chevron_right) : SizedBox.shrink(),
+                  );
+                }));
+  }
+}
+```
+
+This imports the <file>motor_screen.dart</file> file into the program and adds logic to check if a {{< glossary_tooltip term_id="resource" text="resource" >}} is "navigable", or, has a screen made for it.
+Motor is currently the only resource that is navigable.
```

Comment:
"currently..." sounds almost as if Viam currently only supports Flutter motor pages, or something. How about rewording, perhaps:
```suggestion
Since you added a screen for motor, motor is a navigable resource.
```